src/components/ChatMap/ChatMap.jsx

```diff
@@ -59,39 +62,59 @@ export default function ChatMap() {
           name: e.target.feature.properties.ADMIN,
           flagCode: e.target.feature.properties.code,
         };
-        dispatch(sendDataCountryToBackend(userId, data));
-        console.log('data to send', userId, data);
+        setSelectedCountry(data.name);
```

Comment:
Total mess. Defiantly should be refactored.

Reply:
Ok, I need your help with it